styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -415,6 +415,9 @@ private void triggerWorkflowInstance()
     final WorkflowInstance workflowInstance = getWorkflowInstance(namespace);
     final ByteString payload = serialize(workflowInstance);
     send(Request.forUri(apiUrl("scheduler", "trigger"), "POST").withPayload(payload));
+    cliOutput.printMessage(CliOutput.CliMessage.create(
```

Comment:
Would it make sense to include the workflow id and parameter in the message?

Perhaps the json output should be more structured?